libraries/AP_Motors/AP_Motors6DOF.h

```diff
@@ -41,6 +41,8 @@ class AP_Motors6DOF : public AP_MotorsMatrix {
     // output_to_motors - sends minimum values out to the motors
     void output_to_motors() override;
 
+    void set_max_throttle(float max_throttle) { _max_throttle = max_throttle; }
```

Comment:
It would be good to add a comment above this function. Sorry but I think we have comments above every other function